src/main/java/com/google/javascript/clutz/ImportRenameMapBuilder.java

```diff
@@ -121,10 +155,13 @@ private static String getGoogModuleId(Node astRoot) {
             variableName = originalName;
           }
 
-          String localSymbolName = buildLocalSymbolName(localModuleId, variableName);
           String exportedSymbolName = buildNamedExportSymbolName(importedModuleId, originalName);
-          importRenameMap.put(localSymbolName, exportedSymbolName);
           importRenameMap.put(variableName, exportedSymbolName);
+          // If we're in a goog scope, there isn't a module id
+          if (localModuleId != null) {
```

Comment:
looks like a duplicated block, can it be outside the if-else?

Reply:
I don't see a way - the two blocks are subtly different.